src/mpi/rma/rmatypeutil.c

```diff
@@ -48,8 +48,10 @@ int MPIR_Compare_equal(const void *a, const void *b, MPI_Datatype type)
 #define MPIR_OP_TYPE_MACRO(mpi_type_, c_type_, type_name_) \
         case mpi_type_:                         \
             if (*(c_type_ *)a == *(c_type_ *)b) \
+            {                                   \
                 return TRUE;                    \
-                break;
+            }                                   \
+            break;
             MPIR_OP_TYPE_GROUP(C_INTEGER)
```

Comment:
Probably an empty line to separate the macro definition from code is a good idea here.

Reply:
Good point. Let me try if that works.